java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java

```diff
@@ -36,6 +47,75 @@ public VirtualObservableReaderImpl(VirtualReaderSession session, String nativeRe
     }
 
 
+    @Override
+    public void addObserver(ObservableReader.ReaderObserver observer) {
```

Comment:
in addObserver(ObservableReader.ReaderObserver observer), the parameter isn't defined as final